vdoctl/cmd/root.go

```diff
@@ -102,19 +102,20 @@ func initConfig() {
 		fmt.Fprintln(os.Stderr, "Using config file:", viper.ConfigFileUsed())
 	}
 
-	if len(kubeconfig) <= 0 {
-		kubeconfig = os.Getenv("KUBECONFIG")
+	// Ignore the config check and client creation if help command is invoked
+	if os.Args[1] != "help" {
```

Comment:
this can be written as 
if os.Args[1] == "help" {
return
}
can you please verify if help for all commands are working without requiring the config for instance
vdoctl deploy
vdoctl delete
vdoctl configure help
vdoctl configure drivers --help